### DISCUSSION TITLE
VueJS not working if tab doesn't have focus?

### DISCUSSION THREAD

**Discussion by rconstantine** [quasarframework/quasar#9003] [CLI - PWA mode]
Not sure if this is solvable or not. I have a timer that should log a user out automatically if they haven't interacted with the page in 20 minutes. It's a HIPAA thing. If the tab has focus, there is no problem. Without focus, the timer runs down, but then just gets stuck at the end or close to the end of the timer and the function call for the logout never happens. Is this a general web issue, or Vue specific, or Quasar specific, or what? Any solution?

**Comment by ontwikkelfabriek** [comment]
@rconstantine 
Here's an article/tutorial to create an idle timer in Vue:
https://jakzaizzat.com/idle-timer-vue/
